_data/operations.yml

```
@@ -278,3 +278,12 @@ members:
   orcid: 0000-0001-7258-9596
   ror: 02catss52
   wikidata: Q57023310
+- affiliation: Université de Sherbrooke, Québec
+  country: Canada
+  github: pfabry
+  groups:
+  - technical
+  name: Paul Fabry
+  orcid: 0000-0002-3336-2476
+  ror: 00kybxq39
+  wikidata: Q2579532
```

```suggestion
  wikidata: Q108555457
```
The other wikidata entries in this file appear to reference persons, not institutions. For consistency, could you kindly use Q108555457 here?

---

I thought this was clear but maybe we should group the name, location, and ROR for the affiliation in a sub-dictionary in a future PR.

---

@pfabry can you confirm/apply this suggestion?